doc/flux_to_redux.md

```diff
@@ -0,0 +1,579 @@
+
+### How is the transition process different with more stores?
+__You should still only have one Redux store.__ If that doesn't sound scary, you're in good shape and everything is great! Some libraries won't have an issue with that, and in that case it's as simple as following the steps in the next section.
+
+If having one store sounds unfeasible or raises a lot of concerns, there's more to talk about. The authors of Redux have said ([1](https://redux.js.org/api/store#a-note-for-flux-users), [2](https://stackoverflow.com/questions/33619775/redux-multiple-stores-why-not)) that more than one store is not needed, and the exceptions are related to performance and not architecture. The concession is that there are supported ways to have multiple stores, but those are to be used as a last resort and should not be part of an initial refactor attempt.
+
+This leads to the difference in the transition process. Not only do the stores have to come together, but componentry needs to reflect that the data is coming from a single source. 
+
+### Advanced Conversion Step by Step
+> These steps are just a set that make sense for general situations and may not make sense for every library. If the path forward is unclear, they can be referred to for guidance but may need adjustment or supplemental steps.
+1. __Diagram store and component relationships.__ While perhaps challenging and time consuming, creating a diagram that illustrates generally which components care about which stores could prove invaluable to planning the update process.
+1. __Break the refactor into groups that includes the stateful layer and the UI layer.__ Are there small chunks of the system that can be updated without touching the messiest knots in the system? If so, identify them. If this step seems particularly challenging or reveals large roadblocks, consider an [Influx architecture](#influx-architecture)
+1. __For each group, refactor all the stores.__ By doing the stores and reducers at the same time, you can completely invalidate a single store at a time. In other words, you can move all the necessary state logic into the new state class and move the state mutation logic into the reducer. Then, the entire Flux store can be deleted.
+
+    If you have already created a state class and a reducer for an earlier store group, continue adding to the same reducer and state class. Worth noting is that these app examples use a very simplistic reducer approach, and understanding the options with reducers may be worthwhile. Because there are multiple ways to implement reducers, it is likely one will make more sense than others.
+1. __Create any new action classes that are necessary.__ After refactoring a Flux store, it should be clear which actions need to be converted to Redux actions.
+1. __Refactor components into connected components.__ Update the component to `UiComponent2`, removing all Flux boilerplate, and wrapping the already declared factory with the `connect` call.
```

Comment:
Again - it seems odd that the UI portion of these refactors lacks any code examples of any kind...